package.json

```
@@ -106,7 +106,9 @@
     "wait-on": "^5.1.0"
   },
   "scripts": {
-    "start": "run-p --race svelte:watch proxy:start electron:start",
+    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=true start:app",
+    "start:public": "RADICLE_UPSTREAM_EXPERIMENTAL=false start:app",
```

```suggestion
    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false start:app",
    "start:experimental": "RADICLE_UPSTREAM_EXPERIMENTAL=true start:app",
```
As a dev I'm used to run `yarn start` when working with the app. I think the initial idea was that by default we devs should see what a normal user would see. If we're working on new features we should be forced to use env variables to enable these unfinished features.

Maybe @cloudhead can confirm if I understood his reasoning correctly?

---

I don't think it makes much sense to have the default developer workflow not being focus on active development. The use case raised by Merle is that she wanted to run the app and see what the general public will see without going on changing unrelated environment variables, which `yarn start:public` is dedicated for. So the default dev workflow is `yarn start` which shows it all as it makes sense to be in development, and `yarn start:public` to have a public preview of the app.

---

> I think it makes sense to be working with what the user will see by default, because it makes it harder to miss things that we end up shipping.

Cool, I actually really like that :) I had misuderstood the poit initially. Thank you both, will adust!